addons/jam/jam_vn/config.cpp

```diff
@@ -0,0 +1,22 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = ECSTRING(jam,component);
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {
+            "cba_jam",
+            "weapons_v_f_vietnam_c"
```

Comment:
Use `loadorder_f_vietnam`
```suggestion
            "loadorder_f_vietnam"
```

Reply:
changed to `loadorder_f_vietnam`